test/e2e/network/proxy.go

```diff
@@ -180,6 +182,8 @@ var _ = common.SIGDescribe("Proxy", func() {
 			framework.ExpectNoError(err)
 			ginkgo.DeferCleanup(e2erc.DeleteRCAndWaitForGC, f.ClientSet, f.Namespace.Name, cfg.Name)
 
+			deploymentConfig := e2edeployment.NewDeployment(service.Name, 1, labels, service.Name, imageutils.GetE2EImage(imageutils.Agnhost), appsv1.RecreateDeploymentStrategyType)
```

Comment:
this looks incomplete, the Replicaset is still created and running but this deployment is not used later

Reply:
you are right, I must have missed something while separating everything to different PRs or something. fixed now.